### DISCUSSION TITLE
请问一下，支持PP-Structure中的关键信息提取吗？

### DISCUSSION THREAD

**Discussion by Wickedness230** [RapidAI/TableStructureRec#96] [Q&A]
如题

**Comment by Joker1212** [comment]
不支持哈，这个仓库只是做表格提取的

**Comment by Joker1212** [comment]
建议可以考虑轻量级llm模型实现任意情况的信息提取

**Reply by Wickedness230** [reply]
这个和rapidtable的区别是什么？

**Reply by Joker1212** [reply]
rapidTable是来源百度pp-structure系列的模型和算法，这边最开始是阿里读光的模型，后来又加了一些东西，两个仓库都是表格识别，可以试试哪个更适合你的场景

**Reply by Wickedness230** [reply]
WiredTableRecognition() 这里面默认用的是PP-OCRv4 ？

**Reply by Joker1212** [reply]
嗯，用的rapidOCR转的onnx，但是输入参数和paddleOCR不一样，可以看RapidOCR的文档了解

**Reply by Wickedness230** [reply]
再问一下，怎么用GPU加速？

**Reply by Joker1212** [reply]
在faq里面写了哈，主要还是OCR耗时明显，可以使用rapidOCR-paddle来gpu加速，表格模型的话因为不是静态输入,onnx加速效果不好，但是cpu推理有线100ms,无线500ms左右
